src/pages/about.tsx

```diff
@@ -1,12 +1,49 @@
-import { Typography } from "@mui/material";
+import { Stack, Typography } from "@mui/material";
 
 import { text } from "~/assets/text/index.js";
+import { styles } from "~/styles/pages/about.js";
+
+const wordsToLink: Record<string, string> = {
+  "Kryha Labs": "https://kryha.io",
+  Aleo: "https://aleo.org",
```

Comment:
Shouldn't these also be a String?

Reply:
These are strings but prettier is removing the double quotes for the words

Reply:
Interesting...